src/beautifiers/csscomb.coffee

```diff
@@ -6,10 +6,10 @@ module.exports = class JSBeautify extends Beautifier
 
   options: {
     # TODO: Add support for options
-    CSS: false
-    LESS: false
-    Sass: false
-    SCSS: false
+    CSS: true
```

Comment:
This is the last thing that needs fixing. Right now it looks like all options are available, when it should be only `configPath` and `predefinedConfig` option. After that, re-run `coffee docs/` to generate the correct documentation with only the allowed options. Thanks!